Templates.sln

```diff
@@ -656,7 +656,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "TimerTrigger-FSharp", "Time
 EndProject
 Global
 	GlobalSection(SolutionProperties) = preSolution
-		HideSolutionJavaScript = FALSE
```

Comment:
Magic of Visual Studio?  That was not me (though hiding javascript is usually not a bad idea).  Do you want me to fix and resubmit?